src/main/java/org/fcrepo/sword/http/SWORDServiceResource.java

```diff
@@ -28,15 +31,25 @@
  */
 @Scope("request")
 @Path("/sword")
```

Comment:
Absolutely. I will change that.

I build this simliar to the OAI provider which utilizes `/oai` without the `fcr` namespace. Is the OAI provider different from a SWORD provider?